crates/chia-consensus/src/spendbundle_validation.rs

```diff
@@ -24,14 +24,14 @@ pub fn validate_clvm_and_signature(
     max_cost: u64,
     constants: &ConsensusConstants,
     height: u32,
-    cache: &Arc<Mutex<BlsCache>>,
 ) -> Result<(OwnedSpendBundleConditions, Vec<PairingInfo>, Duration), ErrorCode> {
     let start_time = Instant::now();
     let mut a = make_allocator(LIMIT_HEAP);
     let sbc = get_conditions_from_spendbundle(&mut a, spend_bundle, max_cost, height, constants)
         .map_err(|e| e.1)?;
     let npcresult = OwnedSpendBundleConditions::from(&a, sbc);
-    let iter = npcresult.spends.iter().flat_map(|spend| {
+    let spends = npcresult.spends.clone();
```

Comment:
I don't think you need to clone the spends here

Reply:
There's an unfortunate move that happens which means we can't return `npcresult` unless we clone it earlier

Reply:
I would expect you the lambda to take the entry by reference, and return a reference